pysolr.py

```diff
@@ -137,10 +137,8 @@ def force_bytes(value):
 def unescape_html(text):
     """
     Removes HTML or XML character references and entities from a text string.
-
```

Comment:
It looks like there were a bunch of whitespace changes like this; I'd prefer to keep the commit as small as possible by omitting this sort of thing.